Sources/KituraSession/SessionState.swift

```diff
@@ -76,6 +76,8 @@ public class SessionState {
             store.save(sessionId: id, data: data, callback: callback)
         } catch(let error as NSError) {
             callback(error)
+        } catch {
+            callback(nil)
```

Comment:
That's possible. Looking at the diffs since 3.0.2 in swift-corelibs-foundation and swift/stdlib/core, I haven't seen anything that would have caused the casting behavior from Error to NSError to have changed.